docs/mkdocs/documentation/install.md

```diff
@@ -2,8 +2,18 @@
 
 ## Using `kubectl`
 
+First install [cert-manager](https://github.com/cert-manager/cert-manager) which is a dependency.
 ```shell
-kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/default
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+kubectl -n cert-manager wait --for=condition=Available deployment \
+	cert-manager \
+	cert-manager-cainjector \
+	cert-manager-webhook
```

Comment:
Why do we need to wait for all 3 deployments?
```suggestion
kubectl -n cert-manager wait --for=condition=Available deployment cert-manager-webhook
```

Reply:
Also, we should wait for the `Issuer` and `Certificate` CRDs to be `Established`.

Reply:
> Why do we need to wait for all 3 deployments?

We need to wait for the webhook cause it is validating the `Certificate` and the `Issuer` objects. I added the other one just in case cuase this is what the [docs](https://cert-manager.io/docs/installation/verify/) says. I prefer to be on the safe side rather than having another unknown issue in the near future. (The slowest is the webhook anyway).

Reply:
> Also, we should wait for the Issuer and Certificate CRDs to be Established.

Is it possible for an operator to be ready if it's CRDs aren't present yet? I see nothing about it in the docs.

Reply:
> Is it possible for an operator to be ready if it's CRDs aren't present yet?

Probably not since it watches them.